README.md

```diff
@@ -65,9 +65,18 @@ The fastest ways to run PostgreSQL with pgvectorscale are:
 
 You can install pgvectorscale from source and install it in an existing PostgreSQL server
 
+1. Ensure the pgvector extension is available:
+
+  ```sql
```

Comment:
The indenting is not working for these steps: https://github.com/timescale/pgvectorscale/blob/1a9c9041104cdabbb2dcfe2c44b79dce1b83f27b/README.md